tests/AllTests.php

```diff
@@ -11,24 +11,12 @@
 
 use PHP_CodeSniffer\Util\Tokens;
 
-if (defined('PHP_CODESNIFFER_IN_TESTS') === false) {
-    define('PHP_CODESNIFFER_IN_TESTS', true);
-}
-
-if (defined('PHP_CODESNIFFER_CBF') === false) {
-    define('PHP_CODESNIFFER_CBF', false);
-}
-
-if (defined('PHP_CODESNIFFER_VERBOSITY') === false) {
-    define('PHP_CODESNIFFER_VERBOSITY', 0);
-}
+require_once __DIR__.'/bootstrap.php';
```

Comment:
If a contributor is smart enough to have their own `phpunit.xml` (and yes, I do have one), they should be smart enough to keep up with what's being changed in the `phpunit.xml.dist` IMHO.